### PR TITLE
[ Annesa ] Admin Dashboard Youth View

### PR DESCRIPTION
**Pull request by annesatran**
-  Populated the Admin Dashboard's Youth view: displays youths with "active" accounts, their survey completion status/survey answers, youth details including basic youth information, their parent, and emergency contacts.
- Currently does not allow admin to make updates to youth information; will open a new branch for that feature.
    - `YouthSurveyModal` and `YouthDetailsModal` are both built on the `BaseModal` component and use the `InfoRow`, `RatingRow`, and `Rating` components.
  ```
  AdminYouth
    |- YouthSurveyModal
    |    L YouthSurveyDisplay
    |        L RatingRow
    |            L Rating
    L YouthDetailsModal 
         L YouthDetailsModal
  ```
- Added a `yssLight` theme. It's the same as the yss theme except the default background is offWhite.
- Added a `BaseModal` component in `@/components` with the following props:
    - `elementId`: a unique key, make sure that this is unique to the component instance
    - `modalTitle`: header title
    - `btnIcon` (optional): icon to display to the right of the label for the button that opens the modal
    - `btnLabel`: the text label on the button that opens the modal
    - `btnStyling` (optional): any additional tailwind/daisyUi class names to style the button, ex. "btn-primary text-dark"
    - `content` (optional): the content that appears in the modal
    - `theme` (optional): the theme to use, ex. "yss" or "yssLight". The theme should default to the theme of modal's parent, this prop is just used to override it and set it yourself if needed.
- Added a `formatPhoneNumber()` utility function that takes a string or number like 1112223333 and turns into a string in the format (111) 222-3333. 
- Added a `no-scrollbar` styling class, this hides scrollbars from a jsx component.

### CODE REVIEW

File: src/app/(dashboard)/admin/youth/YouthDetailsDisplay.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+          <CollapseSection
+            icon={<MdNoFood size={ICON_SIZE} />}
+            title={"Allergies and Dietary Restrictions"}
+            content={
+              <dl className="divide-y">
+                <InfoRow label={"Allergies"} content={youthDetails.allergies}/>
+                <InfoRow label={"Dietary Restrictions"} content={youthDetails.dietary_restrictions}/>
+              </dl>
+            }
+          />
```
Comment (carolinebnguyen) [inline src/app/(dashboard)/admin/youth/YouthDetailsDisplay.js:85]: Small suggestion: Maybe put "none" by default if the youth has no allergies or dietary restrictions

**Review by carolinebnguyen — APPROVED** [review]
Looks really good! Just made a minor suggestion, but the UI looks great as is :)

File: src/app/(dashboard)/admin/youth/YouthDetailsDisplay.js (hunk 1 above)
Comment (annesatran) [reply src/app/(dashboard)/admin/youth/YouthDetailsDisplay.js]: Fixed!

**Review by carolinebnguyen — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**